<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(164, 208, 255);</string>
  </property>
  <widget class="QPlainTextEdit" name="text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>681</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="page_num_lbl">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>610</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="prev_page_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>610</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(216, 248, 255);</string>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_page_btn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>610</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(216, 248, 255);</string>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="book_name_lbl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>661</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Light</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(216, 248, 255);</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_quote_btn">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>660</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(216, 248, 255);</string>
   </property>
   <property name="text">
    <string>Добавить цитату</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
